### PR TITLE
Update building instructions for Windows

### LINKED ISSUE #1443: Update build instructions to the latest versions of Visual Studio and Qt
### Expected Behaviour
Show instructions for building on Windows with the latest versions of Visual Studio and Qt.

### Actual Behaviour
`BUILDING.md` uses Visual Studio 2017 and Qt 5.9 in the Windows section

### Steps to reproduce
See `BUILDING.md`

### PR DESCRIPTION
**Pull request by Thomas1664**
### Description

Fixes #1443 

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update

### How Has This Been Tested?
I updated the building instructions for Windows for use with Visual Studio 2019 and Qt 5.15.2

**Test Configuration**:
* Operating system: <Name, version number>
* Graphics Card: <Manufacturer (likely Intel, NVidia, AMD?), Model (HD, Geforce, Radeon..., with model number), driver version?>

## Checklist:

- [x] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by alex-w — CHANGES_REQUESTED** [review]
Please fix PATH variable

**Comment by gzotti** [comment]
I would not delete the information applicable to MSVC2017. The changes are mostly trivial, but changed/missing URLs are a problem. Stellarium can still be built with Qt5.9. You can just use MSVC2017 and don't need to bother with upgrading. Further relevant bits of information are 

* the ANGLE library which should be taken from Qt5.6, all later versions don't work. (Don't ask us why. Find a solution!)  
* Qt5.15.0 and 5.15.1 have a buggy lconvert and should not be used. Also lconvert on Qt5.15.2 still allocates GBs of memory for translation of a few MBs of strings, if it can get it.

File: BUILDING.md [hunk 1]
```diff
@@ -236,10 +241,15 @@ $ md build
 $ cd build
 $ md msvc
 $ cd msvc
-$ cmake -DCMAKE_INSTALL_PREFIX=c:\stellarium-bin -G "Visual Studio 15 2017 Win64" ../..
```
Comment (gzotti) [inline BUILDING.md:239]: Show both lines! Who would know "Win64" has to be added for 2017?

**Comment by alex-w** [comment]
the ANGLE library from Qt 5.6 may be obtained from https://github.com/Stellarium/stellarium-data/releases/download/qt-5.6/libGLES-x64.zip or https://github.com/Stellarium/stellarium-data/releases/download/qt-5.6/libGLES-Win32.zip

**Review by alex-w — APPROVED** [review]
Looks OK, @gzotti?

**Review by gzotti — APPROVED** [review]
Fine for me.

**Pull request merged**